src/FOM/CoreBundle/Component/ExportResponse.php

```diff
@@ -110,6 +120,17 @@ public function setXls(array &$data){
         $this->setData($output);
     }
 
+     /**
+     *
+     * @param array $data
+     * @return void
+     */
+
+    public function setXlsx(array &$data){
+        $xlsx = SimpleXLSXGen::fromArray( $data );
```

Comment:
Where does `SimpleXLSXGen` come from? It's not imported and there is no file with that name currently present

Reply:
Ah, okay, then nevermind. Just assumed it was a library dependent on Mapbender.